src/transport_ofi.c

```diff
@@ -297,7 +297,15 @@ static inline int allocate_endpoints(struct fabric_info *info)
      * support shmem_wait that blocks on an update arriving from the network */
 #ifndef ENABLE_HARD_POLLING
     info->p_info->caps |= FI_REMOTE_WRITE | FI_REMOTE_READ;
+#else
+    info->p_info->caps &= ~FI_REMOTE_WRITE;
+    info->p_info->caps &= ~FI_REMOTE_READ;
```

Comment:
yeah guys what did you do in commit: 082c6747e89626bb6facc719e63db040b82388e7 when I was out of town? ;) just kidding. Agreed we should just be keeping the caps=FI_RMA for the ep creation regardless